app/assets/stylesheets/general.scss

```diff
@@ -86,7 +85,8 @@ textarea {
   -o-transition: all 0.375s ease-in-out;
 }
 
-input:focus,
+input[type=text]:focus,
+input[type=password]:focus,
```

Comment:
Change to this `input:focus` (for both of them); this was an incorrect merge conflict fix.